pkgs/top-level/all-packages.nix

```diff
@@ -42097,4 +42097,6 @@ with pkgs;
   };
 
   code-maat = callPackage ../development/tools/code-maat {};
+
+  codeium = callPackage ../development/tools/codeium { };
```

Comment:
if you follow the recent pattern https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md you dont need this